### PR TITLE
26 ansible add jenkins server installation

### PR DESCRIPTION
**Pull request by Olyakaya**
final request

### CODE REVIEW

File: ansible.cfg [hunk 1]
```diff
@@ -6,6 +6,13 @@ stdout_callback = yaml
 host_key_checking = False
 ansible_managed = This file is managed by Ansible
   And it was last modified by the user '{uid}'
+remote_user = ec2-user
+private_key_file=/home/ec2-user/.ssh/deploy.pem
+become = true
+become_method: sudo
+become_user: root
```
Comment (anatolek) [inline ansible.cfg:13]: I believe all these options shouldn't be hardcoded here
```suggestion
```

File: roles/jenkins/README.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
+Installs Jenkins CI on RHEL/CentOS
```
Comment (anatolek) [inline roles/jenkins/README.md:1]:
```suggestion
# Installs Jenkins CI on RHEL/CentOS 
```

File: roles/jenkins/README.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+Role was copied from https://github.com/geerlingguy/ansible-role-jenkins
+## Requirements
+
+Requires `curl` to be installed on the server. Also, newer versions of Jenkins require Java 8+ (see the test playbooks inside the `molecule/default` directory for an example of how to use newer versions of Java for your OS).
+
+## Role Variables
+
+Available variables are listed below, along with default values (see `defaults/main.yml`):
+
+    jenkins_package_state: present
```
Comment (anatolek) [inline roles/jenkins/README.md:12]: Please decorate all code blocks as code blocks

```yaml
 jenkins_package_state: present
```

File: roles/jenkins/README.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+    jenkins_proxy_port: ""
+    jenkins_proxy_noproxy:
+      - "127.0.0.1"
+      - "localhost"
+
+If you are running Jenkins behind a proxy server, configure these options appropriately. Otherwise Jenkins will be configured with a direct Internet connection.
+
+
+
+
```
Comment (anatolek) [inline roles/jenkins/README.md:117]:
```suggestion
```

File: roles/jenkins/handlers/main.yaml [hunk 5]
```diff
@@ -0,0 +1,12 @@
+---
+- name: restart jenkins
+  service: name=jenkins state=restarted
```
Comment (anatolek) [inline roles/jenkins/handlers/main.yaml:3]:
```suggestion
  service:
    name: jenkins
    state: restarted
```

File: roles/jenkins/meta/main.yaml [hunk 6]
```diff
@@ -1,16 +1,16 @@
 ---
 galaxy_info:
-  author: {{ cookiecutter.full_name }}
```
Comment (anatolek) [inline roles/jenkins/meta/main.yaml:3]: I don't understand why you moved these templates?
You should have used the cookiecutter tool to generate a basic role.
Please fix it.

**Review by anatolek — CHANGES_REQUESTED** [review]

File: roles/jenkins/meta/main.yaml (hunk 6 above)
Comment (Olyakaya) [reply roles/jenkins/meta/main.yaml:3]: I got files from https://github.com/geerlingguy/ansible-role-jenkins and https://github.com/geerlingguy/ansible-role-java ...
I left meta but changed molecule similar to common role

File: roles/jenkins/README.md (hunk 3 above)
Comment (Olyakaya) [reply roles/jenkins/README.md]: done, please check

**Review by anatolek — APPROVED** [review]

**Pull request merged**